README-CrossDac.md

```diff
@@ -0,0 +1,62 @@
+### Cross OS DAC Readme
+
+This repo is a fork of the primary CoreCLR release/3.1 branch.
+
+It exists to implement the Windows version of the DAC & DBI libraries
+which we use for debugging. It enables examination of Linux core dumps
+on a Windows machine.
+
+It exists as a separate entity, because the original change was too late
+to be included in the 3.1 release, and the change complexity was too
+high to be considered for backport.
+
+#### Servicing Releases
+
+Periodically, Microsoft will release updates for .NET 3.1 servicing.
+These occur most months, typically around the second Tuesday of each
+month.
+
+For each of these servicing releases we need to release a new cross OS
+DAC/DBI set.
+
+##### Release Process
+
+Assumes the user has a clone of the dotnet/coreclr repo.
+
+In the afternoon when a servicing release has occurred.
+- Checkout this release/3.1-crossdac branch of the repo.
+- Run ./crossDacMergeLatestTag.sh <upstream>
+- Create a PR to merge from <upstream>:release/3.1-crossdac-<latestTag>
+  to dotnet/coreclr:release/3.1-crossdac
+  - Review PR looking for changes which could potentially break the Cross
+    DAC/DBI. These would typically be layout changes to structures or the
+    addition of HOST_* conditional code
+  - In the absence of risky changes, merge the commit.
+  - If there are riskier changes extra eyes might be useful to see if the
+    changes are breaking. (Not really expected.)
+- When the PR is merged an internal trial build will be triggered. See
+  https://dev.azure.com/dnceng/internal/_build?definitionId=244&_a=summary.
```

Comment:
This URL points to an internal AzDO instance and most likely won't be helpful to anyone outside msft. I would leave it out.

Reply:
I don't know.  I find it difficult to find these pipelines. We could move this tribal knowledge into the team OneNote.  But then this whole document could/should go there instead.